Sources/ARKit-CoreLocation/Nodes/LocationNode.swift

```diff
@@ -84,6 +84,11 @@ open class LocationNode: SCNNode {
 
     /// The scheme to use for scaling
     public var scalingScheme: ScalingScheme = .normal
+    
+    /// Whether the node should be stacked along the y-axis accordingly with the distance
+    /// When set to true, scaleRealtiveToDistance should be false
```

Comment:
I did some experiments with different `scalingScheme` but honestly I'm not able to understand the logic behind them. I played with threshold and scale values but I cannot see substantial differences.
It is a "should" because it is not a breaking effect, but it can produces visual unwanted effects. I still need to investigate the usage of `scalingRelativeToDistance` because it is too strong for my purposes.
(Typo fixed)